spec/rubocop/cop/factory_girl/dynamic_attribute_defined_statically_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::FactoryGirl::DynamicAttributeDefinedStatically do
+  subject(:cop) { described_class.new(config) }
+
+  let(:config) { RuboCop::Config.new }
+  let(:bad) { <<-RUBY }
+    FactoryGirl.define do
+      factory :post do
+        status [:draft, :published].sample
+        published_at 1.day.from_now
+        created_at 1.day.ago
+      end
+    end
+  RUBY
+  let(:good) { <<-RUBY }
+    FactoryGirl.define do
+      factory :post do
+        published_at { 1.day.from_now }
+        created_at { 1.day.ago }
+        status :draft
+        comments_count 0
+        title "Static"
+        description FFaker::Lorem.paragraph(10)
+        tag Tag::MAGIC
+      end
+    end
+  RUBY
+
+  it 'registers an offense for offending code' do
+    inspect_source(cop, bad)
+    expect(cop.offenses.size).to eq(3)
```

Comment:
use the expect_violation and expect_no_violations helpers. There are very convenient and make tests very descriptive